babel.config.js

```diff
@@ -48,7 +45,7 @@ CONFIG.es5 = Object.assign({}, CONFIG.default, {
 
 CONFIG.cover = Object.assign({}, CONFIG.default);
 // constant inlining seems to cause problems for nyc
-CONFIG.cover.plugins = ['version-inline', 'istanbul'];
+CONFIG.cover.plugins = ['istanbul'];
```

Comment:
No version, at least on core library?

Reply:
Why do we need this? `deck` and `luma` expose global objects for inspection in the console. For the loaders the version is already defined in package.json.

Reply:
Context is this adds additional overhead to publishing (build needs to be run before publish for dist testing then rerun after `lerna publish` to account for the version bump).